pkgs/games/liberation-circuit/default.nix

```
@@ -0,0 +1,71 @@
+{ stdenv, lib, fetchFromGitHub, fetchurl, cmake, git, makeWrapper, allegro5, libGL }:
+
+stdenv.mkDerivation rec {
+  pname = "liberation-circuit";
+  version = "1.3";
+
+  src = fetchFromGitHub {
+    owner = "linleyh";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "BAv0wEJw4pK77jV+1bWPHeqyU/u0HtZLBF3ETUoQEAk=";
+  };
+
+  patches = [
+    # Linux packaging assets
+    (fetchurl {
+      url = "https://patch-diff.githubusercontent.com/raw/linleyh/liberation-circuit/pull/44.patch";
+      sha256 = "0rvbfvnp3m2yj0mi3g2v2nyrdjj9xr0bzwd8bz20bg2shd4vnj85";
+    })
```

Fetching GitHub Pull Requests should be avoided because they are not immutable and can change e.g. on a force push. This would invalidate the hash and break the build. Since this PR was already merged the merge commit can be fetched instead.

```suggestion
    (fetchurl {
      url = "https://github.com/linleyh/liberation-circuit/commit/72c1f6f4100bd227540aca14a535e7f4ebdeb851.patch";
      sha256 = "0sad1z1lls0hanv88g1q6x5qr4s8f5p42s8j8v55bmwsdc0s5qys";
    })
```

---

Thanks for the tip! I've applied this